test/common/testlib.py

```diff
@@ -910,6 +910,9 @@ def check_journal_messages(self, machine=None):
             self.allowed_messages.append('audit: type=1400 audit(.*): avc:  denied  { mount } for .* comm="find" .*pcp_pmlogger_t.*')
             # HACK: https://bugzilla.redhat.com/show_bug.cgi?id=1662866
             self.allowed_messages.append('audit: type=1400 audit(.*): avc:  denied  { execute } for .* comm="which" .*pcp_pmlogger_t.*')
+            # HACK: https://bugzilla.redhat.com/show_bug.cgi?id=1406979 - the bug is for open operation on a file of the proc directory so it should be identical
```

Comment:
This is an old bug from Fedora 25 in 2017 which got fixed long ago. Does this need to be reopened or a new one?

Reply:
In the BZ which is indeed closed and old It's mentioned there that the fix it's to dontaudit these accesses.
So the warning is supposed to remain there unless we will have AVCRules specifying to don't audit these. Or are these rules builtin to the selinux package?
/me needs to read selinux docs

Reply:
Ok, so after a bit reading I found out what they meant in the BZ. Solution was so manually configure auditd to not audit these messages. So just ignoring these logs here sounds reasonable to me. It's same like configuring auditd to not print them at all.

Reply:
Ack, makes sense; the comment could clarify this a bit. Thanks!

Reply:
ok!

Reply:
I added a comment and added to the https://github.com/cockpit-project/cockpit/pull/11272#issuecomment-470498479 since it has exactly the same problem.

I guess that one will be merged faster to I will rebase this one.